blocks/search-results/search-results.js

```diff
@@ -72,11 +72,14 @@ async function searchPages(placeholders, term, page) {
   const curPage = result.slice(startResult, startResult + resultsPerPage);
 
   curPage.forEach((line) => {
+    console.log(line);
+    let searchTitle;
     const res = document.createElement('div');
     res.classList.add('search-result');
     const header = document.createElement('h3');
     const link = document.createElement('a');
-    const searchTitle = line.pagename || line.breadcrumbtitle || line.title;
+    if (line.type === 'Newsroom') searchTitle = line.breadcrumbtitle || line.title || line.type;
```

Comment:
Didn't get this change, why we need to make this special check

Reply:
@jindaliiita  - we need this since for newsroom type, there are certain things that needs to be populated in the search. That's why. There were couple of instances where line.breadcrumbtitle was empty and hence there was nothing coming up to click. That's why we put line.breadcrumbtitle || line.title || line.type.